tests/robot/CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+
+
+To run the robot tests in `Docker`:
+
+```sh
+
+git clone git@github.com:uillinois-community/uillinois-community.github.io.git
+cd uillinois-community.github.io/tests
+docker run -v ${PWD}/tests/robot/reports:/opt/robotframework/reports:Z -v ${PWD}/tests/robot:/opt/robotframework/tests:Z ppodgorsek/robot-framework:latest
```

Comment:
@edthedev I get a missing directory error when I run this command: `[ ERROR ] Parsing '/opt/robotframework/tests' failed: File or directory to execute does not exist.`

Reply:
@areynold Good catch, thanks! 
I switched from cd-ing to the 'tests' folder to just running the container from the root of the repository clone location and missed correcting the 'cd' command. I've pushed a fix now.
I've pushed a correction.